app/helpers/home_helper.rb

```diff
@@ -21,7 +21,7 @@ def account_link_to(account, button = '', path: nil)
                         end
                     end
                   else
-                    link_to(path || ActivityPub::TagManager.instance.url_for(account), class: 'account__display-name') do
+                    link_to(path || "@#{account.acct}", class: 'account__display-name') do
```

Comment:
I would suggest the following instead:
```suggestion
                    link_to(path || web_url("@#{account.pretty_acct}"), class: 'account__display-name') do
```

But I worry this is going to make things more painful for moderators investigating a suspended remote account, as this is going to replace the link with the local view, which won't show anything since the account is suspended.

Reply:
Thanks, I was hoping that there'd be a better way to create this link!

Do you think it'd be easy enough to check if the account is suspended, and if so use a remote link?

Otherwise, I'm also happy to only change the link on the "Follows and Followers" page as that's most likely to be used by non-moderators.